contracts/leaser/src/contract.rs

```diff
@@ -17,7 +17,9 @@ use crate::{
     state::{config::Config, leases::Leases},
 };
 
-const CONTRACT_VERSION: Version = 1;
+// version info for migration info
+// const CONTRACT_STORAGE_VERSION_FROM: VersionSegment = 0;
+const CONTRACT_STORAGE_VERSION: VersionSegment = 0;
```

Comment:
We agreed on keeping the `CONTRACT_VERSION` not only its component

Reply:
I now understand what you mean and I have tried doing that in the beginning. I wasn't able to get it to compile within a constant context. It thrown an error that the computations exceeded const-eval limits.

Reply:
The only reliable way I can think of doing that is to instead write a procedural macro, which I am not confident in being able to do.

Reply:
no, let's not complicate going with that new macro .... then we can convert the existing macro `parse_.....` into `version!` taking the storage constant as a parameter

The end design goal is to have the `version` lib API deal with the `Version` entity, not with its two components separately, as that is now.

Reply:
Addressed.